pinpoint-web/.env

```diff
@@ -7,7 +7,7 @@ PINPOINT_WEB_NAME=pinpoint-web
 
 SERVER_PORT=8079
 #zookeeper information required
-PINPOINT_ZOOKEEPER_ADDRESS=
+PINPOINT_ZOOKEEPER_ADDRESS=zoo1
```

Comment:
also no need to change, same with collector

Reply:
Oh, Okay I'll delete it.